vortex-expr/src/operators.rs

```diff
@@ -18,6 +18,48 @@ pub enum Operator {
     Or,
 }
 
+#[cfg(feature = "proto")]
+mod proto {
+    use vortex_error::{VortexError, VortexResult};
+    use vortex_proto::expr::BinaryOp;
+
+    use crate::Operator;
+
+    impl TryFrom<Operator> for BinaryOp {
+        type Error = VortexError;
+
+        fn try_from(value: Operator) -> VortexResult<Self> {
+            Ok(match value {
```

Comment:
Why `try_from` if the conversion can't fail?

Reply:
why indeed, I removed this